lib/crp08.py

```diff
@@ -235,6 +235,17 @@ def __init__(self, key):
 		# Binary data
 		self.ecu_data = None
 
+	def __eq__(self, other):
```

Comment:
simple data comparison function.

Reply:
Little bug: self.xtea == other.xtea_salt
And member xtea is an object instance, so the xtrea should also implements a __eq__ function.

Reply:
ah. yes! good catch.

Reply:
done and done